src/Network/Network/ChangeLog.md

```diff
@@ -45,6 +45,11 @@
         - Set-AzureRmExpressRouteConnection : added parameter EnableInternetSecurity
 * Fix required subnet with name AzureBastionSubnet in `PSBastion` can be case insensitive
 * Support for Destination FQDNs in Network Rules and Translated FQDN in NAT Rules for Azure Firewall
+* Add support for ThreatIntelWhitelist property for AzFirewall
```

Comment:
Could you please move this a few positions higher? Let's say after "New cmdlets to support the azure firewall policy"?
I'm concerned that we have a couple of PRs that add their change at the end of the list, so when one is merged, the others will have merge conflicts. Since order in changelog doesn't matter, please add your entry somewhere inbetween.

Reply:
changed